packaging/README.md

```diff
@@ -0,0 +1,481 @@
+# OpenPrinting CUPS: Snap and OCI Image
```

Comment:
Since the packaging directory is about more than just making Snap or OCI images, this should either have a different name or mention the EPM list and RPM spec files as well. Ditto for the TESTING.md file.

Reply:
Yes, that is right, the `README.md` at the top of `packaging/` should cover all contained packaging methods. Or one makes subdirectories, `packaging/rpm`, `packaging/epm`, `packaging/snap`, ... Each one having their own `README.md` files then.